packages/flutter/lib/foundation.dart

```diff
@@ -39,9 +39,11 @@ export 'src/foundation/change_notifier.dart';
 export 'src/foundation/collections.dart';
 export 'src/foundation/consolidate_response.dart';
 export 'src/foundation/constants.dart';
+export 'src/foundation/core_stub.dart'
```

Comment:
Per offline discussion I would suggest changing this to `export 'src/foundation/core.dart';`, and renaming all these platform specific implementations so the filename starts with an `_`. The contents of the `src/foundation/core.dart` file would then just export those, like this:

```
export '_core_stub.dart'
  if (dart.library.html) '_core_web.dart'
  if (dart.library.io) '_core_io.dart';
```

Reply:
I like this approach